### ISSUE TITLE
How to make join on diffent column names?

### ISSUE THREAD

**Issue by mark-ambrazhevich-spiral** [the-tinderbox/ClickhouseBuilder#25]
How to make join on different column names?
```select * from t1 all left join t2 on t1.v =t2.viewId;```

**Comment by evsign** [comment]
For now this types of joins is not supported. I intend to make this possible some time later.

Just as a quick and dirty solution you may create a subquery for join where you can alias needed column for join with needed name.

**Comment by mark-ambrazhevich-spiral** [comment]
Will you accept pull request if I implement this feature?

**Comment by evsign** [comment]
Of course! If this will be test covered and well documented, i will be very glad to accept this!)

**Comment by evsign** [comment]
Also expected that with this will come complete rework of joins "interface" accordingly to pretty big changes in clickhouse.

**Comment by mark-ambrazhevich-spiral** [comment]
Ok, thank you, I'll have a look.

**Comment by mark-ambrazhevich-spiral** [comment]
@evsign , I made pull request, please check.
ps. This is my second github profile.

**Comment by lopn** [comment]
@evsign Is there any follow-up arrangement for this feature? I want to use it
